### ISSUE TITLE
All dropdowns in pagination bar not able to open options via keyboard

### ISSUE THREAD

**Issue by sandarde** [IBM/carbon-components-angular#2285]
## Detailed description

All dropdowns in the pagination bar are not able to open options via the keyboard enter key

## Steps to reproduce the issue
1. Focus on the Item per page dropdown
2. Try to open using a keyboard by hitting the enter key

Expected: Should open dropdown options and after selecting using the up-down key if the user hit enter then needs to set that value.
Actual: Nothing happens by hitting the enter key. 

## Additional information

* Screenshots or code
![image](https://user-images.githubusercontent.com/104478003/195014179-ac2f3fe4-48f0-4c9a-9125-8f6fc2e9ec97.png)

* Notes

**Comment by kush-savani** [comment]
Hello @Akshat55,  According to my investigation, Here we use `select` tag [(Ref)](https://github.com/IBM/carbon-components-angular/blob/3694089d6af3a8b160b6d12159deca6110a7519c/src/pagination/pagination.component.ts#L72-L83). Which is not support open `option` list on Enter key press.

We can override this behaviour with `event.preventDefault();`. but i'm not sure about this solution. should we implement this ?

**Comment by Akshat55** [comment]
Hi @kush-savani & @sandarde 

I have just tested the [pagination story](https://angular.carbondesignsystem.com/?path=/story/components-pagination--basic) and they are keyboard operable. I tabbed to the select & pressed `space` to open them. I can then use my `up` & `down` arrow keys to go through the options. 

The `enter` key is not the appropriate key for opening the dropdown. The default behavior which is implemented by the browser is to use `space` for opening the select dropdown list.

This can also be observed in the [carbon react storybook](https://react.carbondesignsystem.com/?path=/story/components-pagination--default).
